pages/strategy.md

```diff
@@ -0,0 +1,73 @@
+## Introduction
+
+Exercism is a large complex project with thousands of contributors working across dozens of projects, all trying to achieve ambitious goals. It would quickly become overwhelming if we were all working towards making Exercism perfect, so to keep things manageable, we periodically determine the critical next step for Exercism to achieve its goals and then plan that into a strategy.  We present this using the OST framework - defining an **Objective**, which will aim to achieve with a **Strategy**, implemented by a series of **Tactics**.
+
+Our _current_ objective is to make Exercism scalable. 
+
+We expect our next three objectives to be:
+- to make Exercism financially sustainable
+- to make Exercism more accessible to beginners
+- to intentionally grow Exercism for the first time
+
+## Objective
+
+**Make Exercism scalable**
```

Comment:
```suggestion
###Make Exercism scalable
```
Later in the DOC you you switch to this pattern from `**<say somthing>**`.

Reply:
@rpottsoh Thanks :) Yeah - this was deliberate. The reason is that there's only one objective and only one strategy, so visually having the two headings look a bit "off" to me. Once it's merged on the website, I'll check again to see if I think that makes sense or whether I should shift to what you suggest.